src/widgets/study-with-us/ui/study-with-us.module.scss

```diff
@@ -0,0 +1,77 @@
+.stud-with-us {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+
+  @include media-laptop {
+    gap: 48px;
+  }
+}
+
+.study-wrap {
+  display: flex;
+  flex-direction: row;
+  gap: 50px;
+  align-items: flex-start;
+  justify-content: space-between;
+
+  @include media-tablet-large {
+    flex-direction: column;
+    gap: 40px;
+  }
+}
+
+.study-general {
+  width: 640px;
+  text-align: left;
```

Comment:
It's redundant because it's defined in `container`

Reply:
Deletede